### DISCUSSION TITLE
How to open MySQL terminal?

### DISCUSSION THREAD

**Discussion by DavidDivellla** [community/community#54140] [Programming Help] [answered]
### Select Topic Area

Question

### Body

Now I am trying to manage my MySQL database using MySQL terminal window. But I can not find the way,

**Comment by GithubRealFan** [comment, marked as answer]
You can install Navicat premium and manage your MySQL database easily.

**Comment by GithubRealFan** [comment]
You can also type the following command to log in to your MySQL server. mysql -u username -p
Here "username" is your MySQL username.
